lib/iris/tests/__init__.py

```diff
@@ -753,6 +754,21 @@ def tearDown(self):
             plt.close('all')
 
 
+class TestGribMessage(IrisTest):
+
+    def assert_grib_message_contents(self, filename, contents):
+        """
+        Evaluate whether all messages in a GRIB2 file contains contents:
+        an iterable of keys and values.
```

Comment:
@marqh Has this gone stale? Not quite sure what `an iterable of keys and values` applies to ...

Perhaps you could break this down in to the usual doc-string format, see `IrisTest.assertCDL`, to describle `filename` and `contents`